### ISSUE TITLE
DmBotVerification

### ISSUE THREAD

**Issue by DokkixOkuu** [Vencord/plugin-requests#901]
### Plugin Description

The option to remove the stupid bot verification that says "Wait! Are you human?" for when you DM too many people in a short period of time, its just terribly inconvenient when you like to do trades in games. The time this popup lasts is completely random. 
![image](https://github.com/user-attachments/assets/d72db685-23b9-44c6-a61d-398aea3b92ce)

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by AushevAhmad** [comment]
Is this even possible to remove? Like, isn't sort of server sided as protect against bots?

**Comment by MrDiamondDog** [comment]
> Is this even possible to remove? Like, isn't sort of server sided as protect against bots?

It might be, someone would have to look into it

**Comment by Luna-devv** [comment]
This is api gated, every (correctly) implemented CAPTCHA checks server side, this is to prevent automated scripts from DMing/…. Removing this is not possible, and even if, it will get you (very likely) terminated

**Comment by Sqaaakoi** [comment]
Sounds like you want a plugin that bots the [hCaptcha accessibility cookie](https://www.hcaptcha.com/accessibility)<sub>, which is technically against their ToS but at this point violating that is at the same moral standard as blocking ads on YouTube due to the fact they obviously are using the CAPTCHA results for **blatant** AI training.<sub>

Back to Discord and how they use CAPTCHAs; if you are encountering this that frequently it really sounds like the amount of DMs you are sending random users is enough for Discord to think you're a bot, and considering that, I would highly bet the DMs you are sending are spam.

In conclusion: You are likely doing something wrong if you need a CAPTCHA bypass in Discord. CAPTCHA bypass solutions are usually more useful as a web extension for general browsing.

**Comment by Luna-devv** [comment]
hCaptcha gave up on Ai training, in fact, their CAPTCHAs are Ai generated. reCaptcha trains Ai’s with it.

**Comment by Sqaaakoi** [comment]
> hCaptcha gave up on Ai training, in fact, their CAPTCHAs are Ai generated.

that is a better way of describing it, idk if they still do use it for training or not
